chat/src/main/kotlin/rys/ajaxpetproject/chat/infrastructure/adapter/InitialStateEventLoader.kt

```diff
@@ -0,0 +1,8 @@
+package rys.ajaxpetproject.chat.infrastructure.adapter
+
+import reactor.core.publisher.Flux
+import rys.ajaxpetproject.request.message.subscription.proto.EventSubscription
+
+interface InitialStateEventLoader {
```

Comment:
why do you need this interface?

Reply:
After subscription I need to: 
1) Load Initial state
2) Start delivering new Messages that comes 

Earlier I had 1 EventService that done 2 this tasks. As for me its ruins Single Responsibility so I've splatted this service on 2. 

So, in order to DIP I've created Interface which says that whenever who implement it can load initial state

Reply:
deleted